package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
+    "android": "npm run compile && react-native run-android",
```

Comment:
why did you change it?

Reply:
in order to launch the last changes in the emulator, i need to compile the application first.
Before this change , i was doing some change in the code, and even reloading the application in the emulator, i was not able to see my changes, until i understood...that the application was not compiled!!
If you prefer i can add a target:
"android": "react-native run-android",
"android:debug": "npm run compile && react-native run-android",

Reply:
ok change reverted